api_blueprint.apib

```diff
@@ -296,3 +296,121 @@ Execute a search
             X-Domino-Api-Key: YOUR_DOMINO_API_KEY
 
 + Response 200 (application/json)
+
+## Group Models
+
+These APIs allow you to interact with your project's models.
+
+### Get models [GET /v1/projects/{username}/{project_name}/models]
+
+Get a list of models published by a project.
+
++ Parameters
+    + username: `nick` (string, required) - The project owner's username.
+    + project_name: `winequality` (string, required) - The project name.
+
++ Request
+
+    + Headers
+
+            X-Domino-Api-Key: YOUR_DOMINO_API_KEY
+
++ Response 200 (application/json)
+
+    + Attributes
+
+        + objectType: `list` (string) - The type of object the API returned.
+        + data (array) - Array of models.
+            - (object)
+                + id: `57dafe43317467e7b0ef5276` (string) - The unique ID for the run.
+                + name: `Wine Quality Model` (string) - The model name.
+
+### Publish a model [POST /v1/models]
+
+Publish a new model.
+
++ Request
+
+    + Attributes (Model Create)
+
+    + Headers
+
+            X-Domino-Api-Key: YOUR_DOMINO_API_KEY
+
++ Response 200 (application/json)
+
+    + Attributes
+
+        + id: `5a73a6b86278f5c536008832` (string) - The unique ID for the new model.
+
+### Publish a model version [POST /v1/models/{model_id}/versions]
+
+Publish a new model version.
+
++ Parameters
+
+    + model_id: `5a73a6b86278f5c536008832` (string, required) - The unique id of the model to version.
+
++ Request
+
+    + Attributes (Model Base)
+
+    + Headers
+
+            X-Domino-Api-Key: YOUR_DOMINO_API_KEY
+
++ Response 200 (application/json)
+
+    + Attributes
+
+        + id: `5a73a72189c5b2635de7ac5d` (string) - The unique ID for the new model.
+        + number: `2` (number) - The new version number.
+
+## Group Environments
+
+These APIs allow you to interact with your environments.
+
+### Get environments [GET /v1/environments]
```

Comment:
does this get v1 environments or v2 environments? i'm assuming we should be using something to the effect of `/v2/environments` since default environments are v2.